addons/point_of_sale/static/src/customer_display/customer_display.xml

```diff
@@ -35,6 +42,9 @@
                         </div>
                     </div>
                 </div>
+                <div t-attf-class="d-flex justify-content-center {{(order.amount and !order.finalized)? 'bg-view':''}}">
```

Comment:
```suggestion
                <div class="d-flex justify-content-center" t-att-class="{ 'bg-view': order.amount and !order.finalized }>
```

But wait, that is this `bg-view` class? I can't find its definition.

Reply:
Hey @caburj ,
It was used earlier in the code for a white background, so I just use it for that thing. Should I change this to "bg-white".

Reply:
If it works, you can keep it.